arch/arm64/boot/dts/aries/msrz-smarc-common.dtsi

```diff
@@ -71,6 +71,9 @@
 		regulator-max-microvolt = <3300000>;
 		gpios-states = <1>;
 		states = <3300000 1>, <1800000 0>;
+		gpios = <&pinctrl RZG2L_GPIO(39, 1) GPIO_ACTIVE_HIGH>;
```

Comment:
You shouldn't add board-specific ping in the common file. Instead, extend the node in the som-specific part or event board-specific part.

Reply:
I've moved it to the board specific file, please take a look.